src/components/flat-table/flat-table.stories.mdx

```diff
@@ -1142,6 +1050,53 @@ of the button control.
   </Story>
 </Preview>
 
+## Table Cell Sizes
+
+<Preview>
+  <Story name="sizes">
+    <div>
+      {["compact", "small", "medium", "large"].map(size => <Box mb="3">
```

Comment:
There is no margin, I tried editing the mb prop but couldn't figure it out 
![image](https://user-images.githubusercontent.com/72866515/103637048-c8c93f00-4f42-11eb-92e5-de94b65e64aa.png)

Reply:
looks like Box was not imported here, it should work now